### ISSUE TITLE
Use of repeats in classification 

### ISSUE THREAD

**Issue by swarbred** [EI-CoreBioinformatics/minos#18]
Hi @cschu 

I'm not sure using repeats to define the TE related genes is implemented in GMC. I'm running your test set with your repeat file added but my run with real data did not classify TE genes and looking back at earlier runs I don't think I've tested this. 

generate_metrics.py 
```
("repeat", {"metrics": ["nF1", "jF1", "eF1", "aF1"], "parser": None})
```
would seem to indicate you are not parsing this and I don't think we generate nF1 etc as this is not via mikado compare. @gemygk can confirm the process. To date we have only used the repeats in the classification  stage to define biotype transposable_element_gene  not for the fragmentary checks or scoring (currently gmc creates a repeat section in the scoring file). I would add the repeat metric to the metrics matrix but not add it to the mikado scoring config. That way the metric exists i.e. you could manually add this to the config (e.g. to use in the requirements to exclude TE genes) but it's not being used as default (i've been setting  the multiplier as 0 and not_fragmentary_min_value as 1, which effectively turns this off, as it doesn't make sense to use this to score models).

The metric we generate is a single breadth of coverage of each model by all (i.e. overlapping) repeats. I think Gemy is just using bedtools for this. This is then used to define transposable_element_gene and there is a simple coverage cutoff for this (which as user you want to control via the gmc config).

David

**Comment by gemygk** [comment]
cc @swarbred 
We need to make input repeats GFF file format more generic than being specific to RepeatMasker.
When using other tools like [RED](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-015-0654-5), which does not generate a standard GFF3 or a BED output, in which case, we will be creating a GFF3 file in `match->match_part` format which could be used by Minos.

**Comment by swarbred** [comment]
Yep I'm a little confused as this was originally working with match match_part and changed to accommodate the repeatmasker input but I assumed this was in addition to and not to replace.

**Comment by gemygk** [comment]
Fixed in PR #62
